datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxUpgradeEndpoint.java

```diff
@@ -47,5 +47,6 @@ SdxUpgradeResponse upgradeClusterByCrn(@PathParam("crn") String crn,
     @Path("/internal/upgrade_ccm")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "Upgrade Cluster Connectivity Manager", nickname = "upgradeCcm")
-    SdxCcmUpgradeResponse upgradeCcm(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @QueryParam("environment") @NotEmpty String environmentCrn);
+    SdxCcmUpgradeResponse upgradeCcm(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @QueryParam("environment") @NotEmpty String environmentCrn,
```

Comment:
I wonder if it would be more consistent to accept cluster CRN instead of the environment one. Not sure which convention is the more typical.

Reply:
I am not sure that Environment service knows about an environment's data lake CRN

Reply:
Okay, let us keep it this way for now. :+1: